### PR TITLE
feat(app): protocol setup index hifi design

### PR DESCRIPTION
**Pull request by ewagoner**
# Overview

This PR applies the hifi styles to the protocol setup index page.

Designs here: https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/Release%3A-Opentrons-Flex-Touchscreen-Designs?type=design&node-id=6839-89533&t=THfcHXV56l3UZnbH-0

Closes RCORE-754

# Test Plan

Manually tested on sim and devkit. Unit tests changed to match new styles.

# Changelog

- Fixed existing broken unit test
- Re-imported back and more icons from Figma to include the frame
- Imported run icon button svgs from Figma
- Applied hifi styling

# Review requests

Please verify page matches the designs. I've also attached a video of several representative protocols.

# Risk assessment

Low

https://user-images.githubusercontent.com/2960/236099073-23818664-847c-4aa9-9c27-9577abe24af1.mov

### CODE REVIEW

File: app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx [hunk 1]
```diff
@@ -109,12 +102,13 @@ function ProtocolSetupStep({
             {subDetail}
           </StyledText>
         </Flex>
-        <Icon name="chevron-right" size="3rem" />
+        <Icon marginLeft="8px" name="more" size="3rem" />
```
Comment (koji) [inline app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx]:
```suggestion
        <Icon marginLeft={SPACING.spacing3} name="more" size="3rem" />
```

File: app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx [hunk 2]
```diff
@@ -146,17 +140,22 @@ function PlayButton({ disabled, onPlay }: PlayButtonProps): JSX.Element {
...
-      width="4.25rem"
+      width="6.25rem"
       disabled={disabled}
       onClick={onPlay}
       aria-label="play"
     >
-      <Icon color={COLORS.white} marginLeft="0.25rem" name="play" size="2rem" />
+      <Icon
+        color={disabled ? COLORS.darkBlack_sixty : COLORS.white}
+        marginLeft="0.25rem"
```
Comment (koji) [inline app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx]:
```suggestion
        marginLeft={SPACING.spacing2}
```

Comment (ewagoner) [reply app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx]: Actually removed the line altogether -- the new icon didn't need it.

File: app/src/molecules/ODDBackButton/index.tsx [hunk 3]
```diff
@@ -23,17 +23,12 @@ export function ODDBackButton(
       gridGap={SPACING.spacing4}
       lineHeight={TYPOGRAPHY.lineHeight48}
     >
-      <Btn
-        onClick={onClick}
-        paddingLeft="0rem"
-        paddingRight={SPACING.spacing5}
-        width="3rem"
-      >
+      <Btn onClick={onClick} paddingLeft="0rem">
```
Comment (b-cooper) [inline app/src/molecules/ODDBackButton/index.tsx]: zero should be unitless here, but also, is this padding value necessary?

Comment (ewagoner) [reply app/src/molecules/ODDBackButton/index.tsx]: It is no longer necessary. Striking.

**Review by koji — APPROVED** [review]
lgtm!

**Pull request merged**